Tests/StructuredQueriesTests/ViewsTests.swift

```diff
@@ -0,0 +1,76 @@
+import Dependencies
+import Foundation
+import InlineSnapshotTesting
+import StructuredQueries
+import Testing
+import _StructuredQueriesSQLite
+
+extension SnapshotTests {
+  @Suite struct ViewsTests {
+    @Test func basics() {
+      let query = CompletedReminder.createTemporaryView {
+        Reminder
+          .where(\.isCompleted)
+          .select { CompletedReminder.Columns(reminderID: $0.id, title: $0.title) }
+      }
```

Comment:
Not sure if the trailing closure is buying us anything, so could alternately do:

```
CompletedReminder.createTemporaryView(
  Reminder
    .where(\.isCompleted)
    .select { CompletedReminder.Columns(reminderID: $0.id, title: $0.title) }
)
```